### PR TITLE
Adding Cmdlet New-AzApplicationGatewayRewriteRuleUrlConfiguration to support url rewrite capability for V2 application gateways

### PR DESCRIPTION
**Pull request by abjai**
## Description

This should be released as a part of January 17 release.

Adding Support for UrlConfiguration under RewriteRuleActionset for Application Gateway Url Rewrite Feature

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

Powershell design review link: Azure/azure-powershell-cmdlet-review-pr#339
Swagger PR: https://github.com/Azure/azure-rest-api-specs/pull/7905

### CODE REVIEW

File: src/Network/Network.Test/ScenarioTests/ApplicationGatewayTests.ps1 [hunk 1]
```diff
@@ -973,6 +973,262 @@ function Test-ApplicationGatewayCRUDRewriteRuleSetWithConditions
 	}
 }
 
+function Test-ApplicationGatewayCRUDRewriteRuleSetWithUrlConfiguration
```
Comment (abjai) [inline src/Network/Network.Test/ScenarioTests/ApplicationGatewayTests.ps1]: Playback can not be done since the manifest for 2019-01-11 is not updated.

**Comment by isra-fel** [comment]
Hi @abjai what did you mean by Jan 17 release? The ongoing release is on 1/28, see our [milestones page](https://github.com/Azure/azure-powershell/milestones)

**Comment by number213** [comment]
Please retarget to `network-december`

**Comment by abjai** [comment]
Rebasing it to network december

**Comment by abjai** [comment]
Raised another PR to rebase to network-december

https://github.com/Azure/azure-powershell/pull/10894/files

**Comment by abjai** [comment]
> Please retarget to `network-december`

Raised another PR https://github.com/Azure/azure-powershell/pull/10894/files

**Comment by abjai** [comment]
> Hi @abjai what did you mean by Jan 17 release? The ongoing release is on 1/28, see our [milestones page](https://github.com/Azure/azure-powershell/milestones)

Yes. I have rebased my branch to network-december. Please look at the PR here https://github.com/Azure/azure-powershell/pull/10894/files

**Pull request closed without merging**